src/pytest_markdown_docs/plugin.py

```diff
@@ -15,20 +16,40 @@ class MarkdownInlinePythonItem(pytest.Item):
     def __init__(
         self,
         name: str,
-        parent: typing.Union["MarkdownDocstringCodeModule", "MarkdownTextModule"],
+        parent: typing.Union["MarkdownDocstringCodeModule", "MarkdownTextFile"],
         code: str,
+        usefixtures: typing.List[str] = [],
     ) -> None:
         super().__init__(name, parent)
         self.add_marker(MARKER_NAME)
         self.code = code
         self.user_properties.append(("code", code))
 
+        self.usefixtures = usefixtures
+        self.add_marker(pytest.mark.usefixtures(*usefixtures))
+
+    def setup(self):
+        def func() -> None:
+            pass
+
+        self.funcargs = {}
+        self._fixtureinfo = self.session._fixturemanager.getfixtureinfo(
+            node=self, func=func, cls=None, funcargs=False
+        )
+        self.fixture_request = FixtureRequest(self, _ispytest=True)
```

Comment:
haha i looked at this code quite a bit but couldn't figure it out – this was definitely less straightforward than I expected